index.html

```diff
@@ -197,71 +172,53 @@ <h2 class="salute">About Myself</h2>
-              ><img class="social" src="SocialMediaIcons\M_icon.png" alt="M"
-            /></a>
+            <a href="#"><img class="social" src="SocialMediaIcons\M_icon.png" alt="M"/></a>
           </li>
           <li id="git_icon">
-            <a href="#"
-              ><img
-                class="social"
-                src="SocialMediaIcons\Github_icon.png"
-                alt="Github_icon"
-            /></a>
+            <a href="#"><img class="social" src="SocialMediaIcons\Github_icon.png" alt="Github_icon"/></a>
           </li>
           <li id="hand_icon">
-            <a href="#"
-              ><img
-                class="social"
-                src="SocialMediaIcons\Hand_icon.png"
-                alt="Hand_icon"
-            /></a>
+            <a href="#"><img class="social" src="SocialMediaIcons\Hand_icon.png" alt="Hand_icon"/></a>
           </li>
         </ul>
         <a class="get_resume_button" href="#">Get my resume</a>
+      </div>
+      <div class="content content_about_self">
         <ul class="content_about_self2">
           <li class="list_info">
             <p>Languajes</p>
-            <img class="arrow" src="arrow_down.svg" />
-          </li>
-          <li class="list_lang">
-            <img class="lang_logo" src="js_logo.svg" />
-            <p>JavaScript</p>
-          </li>
-          <li class="list_lang">
-            <img class="lang_logo" src="html_logo.svg" />
-            <p>HTML</p>
-          </li>
-          <li class="list_lang">
-            <img class="lang_logo" src="css_logo.svg" />
-            <p>CSS</p>
+            <img class="arrow" src="styles-conference/assets/arrow_down.svg"/>
           </li>
+          <ul class="ul_list_info">
+            <li class="list_lang">
+              <img class="lang_logo" src="styles-conference/assets/js_logo.svg"/>
+              <p>JavaScript</p>
+            </li>
+            <li class="list_lang">
+              <img class="lang_logo" src="styles-conference/assets/html_logo.svg"/>
+              <p>HTML</p>
+            </li>
+            <li class="list_lang">
+              <img class="lang_logo" src="styles-conference/assets/css_logo.svg"/>
+              <p>CSS</p>
+            </li>
+          </ul>
           <li class="list_info">
             <p>Frameworks</p>
-            <img class="arrow" src="arrow_left.svg" />
+            <img class="arrow" src="styles-conference/assets/arrow_left.svg"/>
           </li>
-          <img src="Separator.svg" alt="separator" />
+          <img src="styles-conference/assets/Separator.svg" alt="separator"/>
           <li class="list_info">
             <p>Skills</p>
-            <img class="arrow" src="arrow_left.svg" />
+            <img class="arrow" src="styles-conference/assets/arrow_left.svg"/>
           </li>
-          <img src="Separator.svg" alt="separator" />
+          <img src="styles-conference/assets/Separator.svg" alt="separator"/>
         </ul>
       </div>
     </section>

```

Comment:
- [ ] Kindly make sure the Languages, Frameworks & Skills headers have the proper text color as shown in the [Figma design](https://github.com/Mar12358/Portfolio/pull/10/files#diff-4d23d66c0aa65c0d192c66d068a6b247bb993e2e57ca83c5a98317c94f0838ebR394-R397) as it's not black on the design. ✔
- Kindly fix the "Languajes" typo you made.
- Please fix the line image you've placed below each section (Frameworks & Skills) as they're getting misaligned
     ![Inkedfirefox_gqJnRtmzxh](https://user-images.githubusercontent.com/104968236/228641277-39071e34-75a6-40fd-b4fe-07812852c514.jpg)

Reply:
- [ ] This issue is well resolved :heavy_check_mark: